sdk/spring/scripts/replace_artifact.py

```diff
@@ -104,5 +119,45 @@ def replace_artifact(module):
                 file.write(line)
 
 
+def replace_version_by_artifact(module):
+    """
+    Replace version and x-include-update info
+    :param module:
+    :return:
+    """
+
+    ga_ver_dict = config[module]['ga_ver_dict']
+    pom_list = config[module]['pom_list']
+    for pom in pom_list:
+        log.info('Processing file: {}'.format(pom))
+        with in_place.InPlace(pom) as file:
+            line_num = 0
+            for line in file:
+                line_num = line_num + 1
+                for ga_key, version_tuple in ga_ver_dict.items():
```

Comment:
I guess `ga` mean `groupId and artifactId`, but I'm not sure I can remember it in the future. Could you please rename `ga_key` to `key`?